### ISSUE TITLE
Code cleanup for pmp-redesign

### ISSUE THREAD

**Issue by adi130987** [unicef/etools-issues#485]
* agreement amendments, 
 `_canAddAmendments`, remove condition containing `active`

**Comment by daniel-tabarcea** [comment]
We also need to clean-up "active" statuses that the old api used

**Comment by adi130987** [comment]
@daniel-tabarcea please be more specific... add the places where the cleanup is needed in this issue.

**Comment by daniel-tabarcea** [comment]
@adi130987 You're right. I've edited the previous comment and added the exact place where the update is needed
